tasks/arrayToObject.js

```diff
@@ -0,0 +1,21 @@
+// get the old array, store it in json variable
+const json = require('../data/oldCardsArray.json');
+const fs = require('fs');
+
+// new object to write
+const newJson = {};
+
+// iterate through the array
+for (let i = 0; i < json.length; i += 1) {
```

Comment:
Almost all of the ids are in uppercase, but there are a few lowercase ones (e.g.: tt_004).  Would you rather have those at the end, or mixed in with the "normal" ids?